metadata-ingestion/tests/unit/test_usage_common_tmp.py

```diff
@@ -0,0 +1,174 @@
+from datetime import datetime
```

Comment:
Is this added by mistake? Can we merge this with test_usage_common.py?

Reply:
Yeah, it was a mistake. I removed it.